distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+|                +------------+------+                       |
+|                |cDEF        |c369  |                       |
+|                | Operations | Data |                       |
+|/---------------+------------+------+------------+---------\|
+||cDEF           |cDEF               |cDEF        |cDEF     ||
+||  Transformers |                   | Federation | Sources ||
+|+---------------+ Catalog Framework +------------+---------+|
+||cDEF           |                   |cDEF   Eventing       ||
+||   Catalog     |                   +------------+---------+|
+||   Plugins     |                   |cDEF   Resources      ||
+|\---------------+-------------------+----------------------/|
+|                |cDEF               |                       |
+|                | Catalog Provider  |                       |
+|                +-------------------+                       |
+|                |cDEF               |                       |
+|                | Storage Provider  |                       |
+|                \-------------------/                       |
++------------------------------------------------------------+
+....
+
+The Catalog stores and translates Metadata, which can be transformed into many data formats, shared, and queried.
+The primary form of this metadata is the metacard. 
+A `Metacard` is a container for metadata. 
+`CatalogProviders` accept `Metacards` as input for ingest, and `Sources` search for metadata and return matching `Results` that include `Metacards`.
+
+=== Metacards
+
+A metacard is a single instance of metadata in the Catalog (an instance of a metacard type) which
+generally contains general information about the product, such as the title of the product, the product's geo-location, the date the product was created and/or modified, the owner or producer, and/or the security classification. 
+
+==== Metacard Type
+
+A metacard type indicates the attributes available for a particular metacard.
+It is a model used to define the attributes of a metacard, much like a schema.
+
+A metacard type indicates the attributes available for a particular type of data.
+For example, an image may have different attributes than a PDF document, so each could be defined to have their own metacard type.
+
+===== Default Metacard Type and Attributes
+
+Most metacards within the system are created using the default metacard type or a metacard type based on the default type.
+The default metacard type of the system can be programmatically retrieved by calling `ddf.catalog.data.BasicTypes.BASIC_METACARD`.
+The name of the default `MetacardType` can be retrieved from `ddf.catalog.data.MetacardType.DEFAULT_METACARD_TYPE_NAME`.
+
+The default metacard type has the following required attributes.
+Though the following attributes are required on all metacard types, setting their values is optional except for ID.
+
+.[[_metacard_required_attributes]]Metacard Required Attributes
+[cols="4*", options="header"]
+|===
```

Comment:
I believe all Core attributes are required for any metacard type: https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/types/Core.java

Here are the missing `Core` attributes.. was there a reason these were omitted?
`datatype` - https://codice.atlassian.net/wiki/display/DDF/datatype
`description` - https://codice.atlassian.net/wiki/display/DDF/description
`language` - https://codice.atlassian.net/wiki/display/DDF/language
`resource.derived-download-url` - https://codice.atlassian.net/wiki/display/DDF/resource-derived-download-url
`resource.derived-uri` - https://codice.atlassian.net/wiki/display/DDF/resource-derived-uri
`resource-download-url` - https://codice.atlassian.net/wiki/display/DDF/resource-download-url
`source-id` - There has been discussion about this being deprecated. https://codice.atlassian.net/wiki/display/DDF/source-id
`metacard.created`, `metacard.modified`, `metacard.owner`, `metacard-tags` - https://codice.atlassian.net/wiki/pages/viewpage.action?pageId=36995090